tibetegya/Cartale_api
# add code quality tools

#### What does this PR do?
Adds Code quality checks plus the badges to the repo
#### Description of Task to be completed?
- Add code climate
- Add Travis CI
- Add Coveralls
#### How should this be manually tested?
- Navigate to the ReadMe.md file and you should be able to see badges for the listed tools.
- Click on any badge and it should redirect you to the report page.

fixes #23
